<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OcrQueue</class>
 <widget class="QWidget" name="OcrQueue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">OcrQueue</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>queue.title</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="ocr_addImageButton">
          <property name="text">
           <string>queue.addImageButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ocr_removeSelectedButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>queue.removeSelected</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ocr_removeAllButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>queue.removeAll</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="optionsDialogButton">
          <property name="text">
           <string>queue.optionsButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ocr_startButton">
          <property name="text">
           <string>queue.startOcrButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="tableView">
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="format">
          <string notr="true">%v/%m - %p%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>results</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="ocr_acceptSelectedButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>results.acceptSelectedButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ocr_acceptAllButton">
          <property name="text">
           <string>results.acceptAllButton</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="ocr_ignoreValidateCheckBox">
          <property name="text">
           <string>results.ignoreValidate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
